cmd/jimmsrv/service/service.go

```diff
@@ -375,7 +375,7 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 	})
 
 	redirectUrl := p.PublicDNSName + jimmhttp.AuthResourceBasePath + jimmhttp.CallbackEndpoint
-	if !strings.HasPrefix(redirectUrl, "https://") || !strings.HasPrefix(redirectUrl, "http://") {
+	if !strings.HasPrefix(redirectUrl, "https://") && !strings.HasPrefix(redirectUrl, "http://") {
```

Comment:
Does this need changing? Because this would only apply to when it has no protocol?